schema.sql

```diff
@@ -1,5 +1,11 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
+CREATE TABLE animals
+(
+    id integer,
```

Comment:
- [ OPTIONAL] You can make the id column autoincrement and not nullable. You can also make it the primary key for this table. The good thing about doing such is that you won't have to manually populate the `id` column with data. 👍 

`id SERIAL PRIMARY KEY` 👍

Reply:
@Gambit142 Thank you so much I was actually thinking of how to handle that.